### PR TITLE
chore: make rust version for cw as official and inputs grouping

### PR DESCRIPTION
**Pull request by dzmitry-lahoda**
seems all support for python can go as hints to poetry2nix

### CODE REVIEW

**Comment by dzmitry-lahoda** [comment]
question: were to put python package? not sure it is usual package so. should it be in python3Packages somewhere?

**Comment by dzmitry-lahoda** [comment]
in runtime seems cosmpy needs
```
[tool.poetry.dependencies]
python = "^3.8"
ecdsa = "*"
bech32 = "*"
requests = "*"
protobuf = ">=4.21.6,<5.0dev"
grpcio = "==1.53.2"
jsonschema = ">=3.2.0,<5"
python-dateutil = "*"
pycryptodome = "^3.18.0"
googleapis-common-protos = "*"
```
so really somehow need allow to list these defaults modules(from proj), so adding cosmpy leads to adding these (likely via poetry deps or like)

File: flake.nix [hunk 1]
```diff
@@ -227,19 +224,24 @@
...
-    namada-src.flake = false;
+    # Tools
+    cosmwasm-src.url = "github:CosmWasm/cosmwasm/v1.5.0";
+    cosmwasm-src.flake = false;
+
+    beaker-src.url = "github:osmosis-labs/beaker/v0.1.6";
+    beaker-src.flake = false;
+
+    gex-src.url = "github:cosmos/gex/233d335dc9e8c89fb318d1081fae74435f6cac11";
+    gex-src.flake = false;
```
Comment (JonathanLorimer) [inline flake.nix:245]: nice

**Review by JonathanLorimer — APPROVED** [review]

**Comment by dzmitry-lahoda** [comment]
closing, will open new pr

**Pull request closed without merging**